### DISCUSSION TITLE
小提示Tips征集

### DISCUSSION THREAD

**Discussion by ChingCdesu** [MaaAssistantArknights/MeoAsstElectronUI#37] [Ideas]
### 开始征集一些有趣的小提示Tips。不过有以下一些要求
- 尽可能简短，在25字以内
- 只能是文字形式，不能用表情包、图片等形式
- 可以是动态的，随各种因素变化，比如时间变化（“今天周一，剿灭又刷新了哦”）

**Comment by 0osmanthus** [comment]
可以考虑一下提示今天是哪个干员的生日？

**Comment by Tanulav** [comment]
可以考虑来点干员的语音内容？

**Comment by ChingCdesu** [comment]
> 可以考虑来点干员的语音内容？

是玩梗的语音内容吗（

**Comment by Tanulav** [comment]
> > 可以考虑来点干员的语音内容？
> 
> 是玩梗的语音内容吗（

emm，那种关心刀客塔身体的？或者是“你有没有听见....”

**Reply by ChingCdesu** [reply]
半夜了，刀客塔还不能休息哦（坏

**Comment by ABA2396** [comment]
冷知识 比如角色真名；慕斯有两条尾巴；ios端有4月入职的Dr. 等等

**Comment by EightOrange** [comment]
水月肉鸽开了，剧情多这么多拓展是不是可以整点梗（
比如：临光和暇光看的小说和骑士哥的联系；叔叔那个赏金猎人朋友非常强，但是和这位赏金猎人一样强的老哥在骑猎活动被蒂蒂当杂兵刷等等

总之就是可以加点剧情skip党完全不知道，不认真看剧情的玩家也不清楚，只有仔细看的人才知道的梗，最好不太涉及剧情本身

当然如果征得画师们同意的话，或许可以加入水月妈和蒂蒂爹在推特的互动梗之类的外网梗（）

**Comment by bakashigure** [comment]
在实装了!

**Comment by Higher-Stark** [comment]
考虑一下泰拉天气预报吗？
2023年4月23日，炎国局部有寒潮，注意添衣保暖。
也可以玩不正经的，1099年2月30日，泰拉某地突降暴雨，这种

**Comment by Rbqwow** [comment]
“你知道吗：MAA 的 ‘M’ 最开始指的是 ‘Meo’，而不是 ‘Maa’。”
🤨所以说到底是什么时候改的名

**Reply by MistEO** [reply]
NONO, 特地改名就是为了抹掉个人属性（
